mandatory/3-function-output.js

```diff
@@ -8,12 +8,16 @@ function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
+// dd
+
 function concatenate(firstWord, secondWord, thirdWord) {
+  return (`${firstWord} ${secondWord} ${thirdWord}`)
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
 }
 
 /* 
+ 
 ===================================================
 ======= TESTS - DO NOT MODIFY BELOW THIS LINE =====
 
```

Comment:
Hi Danny, please try to add comments for thhis tasks if you get some free time. You can google it or use MDN website as well. Thanks